packages/discord.js/typings/index.d.ts

```diff
@@ -5304,6 +5304,7 @@ export interface ClientOptions {
   allowedMentions?: MessageMentionOptions;
   partials?: readonly Partials[];
   failIfNotExists?: boolean;
+  /** @deprecated In the next release, use {@link ClientOptions.ws.initialPresence} instead */
```

Comment:
By the time users see this, it'll be already that release:
```suggestion
  /** @deprecated Use {@link ClientOptions.ws.initialPresence} instead */
```

Reply:
> By the time users see this, it'll be already that release:

No...? Vlad said we're deprecating in 14 and removing completely in 15.

Reply:
In retrospect though, wsOptions.presence is literally not a thing right now. It's probably best if we started supporting it in 14 right now.

Reply:
This is landing in the next version 14 release. The message they'll see is "In the next release use...", but it should be only "Use".